Author: Simon Elsbrock <simon@iodev.org>
Description: remove unneeded bonsai dependency, require yaml module
 .
 Removes a dependency that is not needed for the build process and also not
 available in Debian. Therefore, not forwarded.
Last-Update: 2013-02-20
Forwarded: not-needed

---

--- a/docs/Rakefile
+++ b/docs/Rakefile
@@ -1,6 +1,4 @@
-require 'bonsai'
-require 'liquid'
-require 'maruku'
+require 'yaml'
 require 'json'
 require 'ronn'
 require 'tempfile'
@@ -24,7 +22,6 @@ module ExtraFilters
   end
 end
 
-Liquid::Template.register_filter(ExtraFilters)
 
 
 task :serve do
